client/src/index.scss

```diff
@@ -1,3 +1,5 @@
+@import '../assets/scss/colors.scss';
```

Comment:
Project has global imports, you should be able to just import `@import 'colors';` instead of using a relative path

Reply:
I see what you're doing with the color now. Let's remove this along with the bottom code.